### PR TITLE
Update README.asciidoc

### PR DESCRIPTION
**Pull request by dr3san**

### CODE REVIEW

File: README.asciidoc [hunk 1]
```diff
@@ -1,2 +1,7 @@
-TODO
-Instructions how to set up and run the code.
+== Instructions
+
+1. Get the required libraries.
```
Comment (Akustav) [inline README.asciidoc:3]: What are the required libraries, a short description would be good. The code has currently a specific python requirement due to the realsense library so this is also a good thing to mention.

File: README.asciidoc [hunk 2]
```diff
@@ -1,2 +1,7 @@
...
+
+2. Run motion.py to test if motors work.
+
+3. Run main.py to see the robot in action.
```
Comment (Akustav) [inline README.asciidoc:7]: Are aspects of the code configurable? How to configure them?

File: README.asciidoc [hunk 3]
```diff
@@ -1,2 +1,7 @@
-TODO
-Instructions how to set up and run the code.
```
Comment (Akustav) [inline README.asciidoc:2]: A short description as to what this code does would also be good.

**Review by Akustav — CHANGES_REQUESTED** [review]

File: README.asciidoc (hunk 1 above)
Comment (dr3san) [reply README.asciidoc]: added list of depencencies and short description

File: README.asciidoc (hunk 2 above)
Comment (dr3san) [reply README.asciidoc]: described, how to configure the code

File: README.asciidoc (hunk 3 above)
Comment (dr3san) [reply README.asciidoc:2]: done

File: README.asciidoc [hunk 4]
```diff
@@ -1,2 +1,31 @@
...
+This code is used for robots participating in robot basketball competions. The competition rules are described here: https://ut-robotics.github.io/robot-basketball-rules/
```
Comment (Akustav) [inline README.asciidoc:1]: A note about the readme structure:

https://ut-robotics.github.io/picr22-home/team_documentation_requirements.html

This file might work as the software readme but the main readme needs to fulfil more conditions. Just so you know

**Review by Akustav — CHANGES_REQUESTED** [review]
This looks like the code readme file. The main readme needs to also be filled

File: README.asciidoc (hunk 4 above)
Comment (dr3san) [reply README.asciidoc:1]: The main readme with game logic documentation etc. is in the documentation branch...

**Comment by dr3san** [comment]
please review

**Comment by ReikoR** [comment]
Write documentation here:
https://github.com/ut-robotics/picr22-team-snax/blob/software/software/README.asciidoc

Then create pull request from `software` branch to `main` branch.

**Pull request closed without merging**